src/main/java/homework_3/ImmutableClass.java

```diff
@@ -11,19 +11,51 @@
 
     private final String string;
     private final int intVal;
+    private final StringBuilder stringBuilder;
 
-    private ImmutableClass(String string, int intVal) {
+    private ImmutableClass(String string, int intVal, StringBuilder stringBuilder) {
         this.string = string;
         this.intVal = intVal;
+        this.stringBuilder = new StringBuilder(stringBuilder);
     }
 
     private ImmutableClass() {
         this.string = "defaultVal";
         this.intVal = 0;
+        this.stringBuilder = new StringBuilder("defaultValue");
     }
 
+    @Override
+    public String toString() {
+        return "ImmutableClass{" +
+                "string='" + string + '\'' +
+                ", intVal=" + intVal +
+                ", stringBuilder=" + stringBuilder +
+                '}';
+    }
+
+    public String getString() {
+        return string;
+    }
+
+    public StringBuilder getStringBuilder() {
+        return new StringBuilder(stringBuilder);
+    }
     private ImmutableClass createNewObj() {
         return new ImmutableClass();
     }
```

Comment:
There's no purpose for this method. Doesn't fit the criteria

Reply:
Yep, should i just make this method as public or this method must accept any values ?